third_party/utils/src/test/scala/tools/cmd/Scala_2_13_9_CommandLineParser.scala

```diff
@@ -0,0 +1,8 @@
+package scala.tools.cmd;
```

Comment:
It might be unlikely but in theory some other compiler plugin might define the same workaround. Have you maybe considered creating a dedicated proxy explicitly for rulesscala to decrease chances of name clash? 

I had this issue recently when starting work on dependency analyzer plugin port to Scala 3. I've defined it there as following: 
For Scala 2.13.10+
```scala
package scala {
  package rulesscala {
    // proxy to private[scala] compiler API
    object Proxy {
      def tokenize(cmd: String): List[String] = sys.process.Parser.tokenize(cmd)
    }
  }
}

package io.bazel.rulesscala.utils {
  trait CompilerAPICompat {
    def tokenize(cmd: String): List[String] = scala.rulesscala.Proxy.tokenize(cmd)
  }
}
```

For other versions: 
```scala
package io.bazel.rulesscala.utils

import scala.tools.cmd.CommandLineParser

trait CompilerAPICompat {
  def tokenize(cmd: String): List[String] = CommandLineParser.tokenize(cmd)
}

```

Then I've redefined `object TestUtil` to extend this trait and used a proxy method instead of depending on compiler internals

Reply:
Oh, this is better. I haven't actually done much Scala in a while, so this approach didn't occur to me, but it's definitely better than monkeypatching `scala.tools.cmd`.

I didn't see your original code in the existing code base, so I literally copied and pasted it into a new commit (crediting you, of course).

If this should live somewhere else, or you're about to check it in somewhere else (a quick `find | xargs grep` didn't turn it up), let me know.